homeassistant/components/isy994.py

```diff
@@ -132,6 +134,20 @@ def _categorize_programs() -> None:
                         PROGRAMS[component].append(program)
 
 
+def _categorize_weather() -> None:
+    """Categorize the ISY994 weather data."""
+    global WEATHER_NODES
```

Comment:
Instead of `global`, can you store it in `hass.data`. It's a dict that stores global data for the duration of hass being on.